tests/docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+services:
```

Comment:
You can [set up nextest](https://nexte.st/docs/configuration/setup-scripts/) to do pre-test setup automatically in addition to having a composefile around.

This may also be necessary if you add more than one test to prevent test concurrency from causing issues.

Reply:
This I'll fix when I do the tests. I'll make a separate PR just for the tests